### PR TITLE
create magazine usign react and split into components

### PR DESCRIPTION
**Pull request by jhonmartinezG**
Just the magazine but now split into components and using props as well.

### CODE REVIEW

File: Sebastian-Martinez/tarea7/portada-react/src/components/ContentCenter/ContentCenter.jsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
+import AlertBox from "../AlertBox/AlertBox";
+import SectionText from "../SectionText/SectionText";
+import SectionTitle from "../SectionTitle/SectionTitle";
+import "./ContentCenter.css";
+
+const ContentCenter = () => {
+  return (
+    <div style={{ display: "flex", justifyContent:"space-between" }} className="content-center">
```
Comment (emolinaab) [inline Sebastian-Martinez/tarea7/portada-react/src/components/ContentCenter/ContentCenter.jsx:8]: @jhonmartinezG , using inline styles is a bad practice please move this to class, also this div should be a section

File: Sebastian-Martinez/tarea7/portada-react/src/components/ContentCenter/ContentCenter.jsx [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+      <div style={{ display: "flex", flexDirection: "column", justifyContent: "space-between", alignItems:"flex-start"}}>
```
Comment (emolinaab) [inline Sebastian-Martinez/tarea7/portada-react/src/components/ContentCenter/ContentCenter.jsx:9]: @jhonmartinezG , using inline styles is a bad practice please move this to class, also this div should be a section

File: Sebastian-Martinez/tarea7/portada-react/src/components/ContentCenter/ContentCenter.jsx [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+          <SectionText
+            text={
+              "The news about the new tour by Miley Cyrus and her what her family think about it "
+            }
+          />
+        </div>
+      </div>
+
+
+      <div className="right" style={{  marginTop:"300px",height:"100%", display: "flex", flexDirection: "column", justifyContent: "center", alignItems: "center"}}>
```
Comment (emolinaab) [inline Sebastian-Martinez/tarea7/portada-react/src/components/ContentCenter/ContentCenter.jsx:40]: @jhonmartinezG , using inline styles is a bad practice please move this to class, also this div should be a section

File: Sebastian-Martinez/tarea7/portada-react/src/components/Header/Header.jsx [hunk 4]
```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import PrincipalTitle from '../PrincipalTitle/PrincipalTitle'
+import TopTitle from '../TopTitle/TopTitle'
+import './Header.css'
+
+
+const Header = () => {
+  return (
+    <div className='header'>
```
Comment (emolinaab) [inline Sebastian-Martinez/tarea7/portada-react/src/components/Header/Header.jsx:9]: @jhonmartinezG you should use header tag

File: Sebastian-Martinez/tarea7/portada-react/src/components/SectionText/SectionText.jsx [hunk 5]
```diff
@@ -0,0 +1,10 @@
+import './SectionText.css'
+const SectionText = ({ text }) => {
+  return (
+    <div className="container-section-text">
```
Comment (emolinaab) [inline Sebastian-Martinez/tarea7/portada-react/src/components/SectionText/SectionText.jsx:4]: @jhonmartinezG this should be a section

**Review by emolinaab — CHANGES_REQUESTED** [review]

File: Sebastian-Martinez/tarea7/portada-react/src/components/ContentCenter/ContentCenter.jsx (hunk 3 above)
Comment (jhonmartinezG) [reply Sebastian-Martinez/tarea7/portada-react/src/components/ContentCenter/ContentCenter.jsx]: fixed in the new commit

File: Sebastian-Martinez/tarea7/portada-react/src/components/Header/Header.jsx (hunk 4 above)
Comment (jhonmartinezG) [reply Sebastian-Martinez/tarea7/portada-react/src/components/Header/Header.jsx]: fixed in the new commit

File: Sebastian-Martinez/tarea7/portada-react/src/components/SectionText/SectionText.jsx (hunk 5 above)
Comment (jhonmartinezG) [reply Sebastian-Martinez/tarea7/portada-react/src/components/SectionText/SectionText.jsx]: fixed in the new commit

File: Sebastian-Martinez/tarea7/portada-react/src/components/ContentCenter/ContentCenter.jsx (hunk 2 above)
Comment (jhonmartinezG) [reply Sebastian-Martinez/tarea7/portada-react/src/components/ContentCenter/ContentCenter.jsx]: fixed in the new commit

**Review by emolinaab — APPROVED** [review]

**Pull request merged**